packages/devextreme/js/__internal/ui/m_drop_down_button.ts

```diff
@@ -412,6 +414,11 @@ class DropDownButton extends Widget<DropDownButtonProperties> {
         at: `${horizontalAlignment} bottom`,
       },
       _wrapperClassExternal: DROP_DOWN_EDITOR_OVERLAY_CLASS,
+      integrationOptions: {
```

Comment:
If we now have 2 places where template is rendered, isn't it more logical to remove this rendering in ddButton and completely delegate it to Overlay, not vice versa (reimplement in ddButton and restrict behavior on nested component)?

Reply:
I think it will be expensive to rework because:
1. DDButton can use a named template, in which case we will have to make sure that the template name is passed to Overlay;
2. By default, List is rendered there (this mechanism will need to be completely transferred and may be more difficult to manage).

The idea is still to completely control the rendering from DropDownButton – this does not contradict the principles.